### PR TITLE
remove dimmer from structureboard

### PR DESCRIPTION
**Pull request by vxsx**
we don't really need extra element for it

### CODE REVIEW

File: cms/static/cms/sass/components/_structureboard.scss [hunk 1]
```diff
@@ -8,19 +8,8 @@
     z-index: z(structure, base);
     width: 100%;
     height: 100%;
+    background: $structure-bgcolor;
 
-    .cms-structure-dimmer {
```
Comment (FinalAngel) [inline cms/static/cms/sass/components/_structureboard.scss]: I also implemented a dimmer for the sideframe, eventually we might want to create a component out of it?

**Comment by FinalAngel** [comment]
LGTM

File: cms/static/cms/sass/components/_structureboard.scss (hunk 1 above)
Comment (vxsx) [inline cms/static/cms/sass/components/_structureboard.scss]: i'm removing it, what are you talking about? :)

Comment (FinalAngel) [inline cms/static/cms/sass/components/_structureboard.scss]: yeah I know, just my thoughts on making it general if we might need it somewhere else (currently only used within the sideframe after this removal)

**Comment by vxsx** [comment]
NB: merging this, let's see if the build will break when this is merged into `preview`

**Pull request merged**